ReactSkia/sdk/NotificationCenter.h

```diff
@@ -62,9 +65,11 @@ class NotificationCenter {
 
         NotificationCenter(const NotificationCenter&) = delete;  
         const NotificationCenter& operator = (const NotificationCenter&) = delete;
+        folly::ScopedEventBaseThread eventNotifierThread_;
     public:
-        NotificationCenter() {
-            NotificationCenter::last_listener = 0;
+        NotificationCenter():eventNotifierThread_("NotificationCenterThread") {
+          NotificationCenter::last_listener = 0;
```

Comment:
can we improve the initialization of last_listener data member.
Either during declaration or as part of initializer list